docs/helm-broker/docs/03-01-create-bundles.md

```diff
@@ -85,6 +90,27 @@ The `plans` directory must contain at least one plan. Each plan must contain the
 
 >**NOTE:** For more information about schemas, see [this](https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md#schemas-object) specification.
 
+## docs directory
+
+In the `docs` directory we specify documentation for the bundle and for the wrapped Helm Chart. You can define a `meta.yaml` file inside the `docs` folder which holds the information on how documentation for a bundle should be uploaded.
+Currently we provide the documentation for bundle using the ClusterDocsTopics because in Kyma the Helm Broker is installed as a ClusterServiceBroker. For more information about ClusterDocsTopics see the following [file](https://kyma-project.io/docs/components/headless-cms/#custom-resource-clusterdocstopic).
+If the `docs/meta.yaml` is specified the Helm Broker tries to create the ClusterDocsTopic for this bundle. Below you can see the example structure of the `meta.yaml` file.
+```
+docs:
+    - template:                     # template describes the ClusterDocsTopic that will be created.
```

Comment:
we provided an example, but where I can learn more information about which fields I can provide?

Reply:
I added a sentence about this below the example